contracts/near/eth-types/src/lib.rs

```diff
@@ -177,6 +177,31 @@ pub type Signature = H520;
 
 #[derive(Debug, Clone, BorshSerialize)]
 #[cfg_attr(not(target_arch = "wasm32"), derive(Serialize, Deserialize))]
+pub struct BlockHeaderBase {
+    pub parent_hash: H256,
+    pub uncles_hash: H256,
+    pub author: Address,
+    pub state_root: H256,
+    pub transactions_root: H256,
+    pub receipts_root: H256,
+    pub log_bloom: Bloom,
+    pub difficulty: U256,
+    pub number: u64,
+    pub gas_limit: U256,
+    pub gas_used: U256,
+    pub timestamp: u64,
+    pub extra_data: Vec<u8>,
+    pub mix_hash: H256,
+    pub nonce: H64,
+    #[cfg(feature = "eip1559")]
+    pub base_fee_per_gas: u64,
+
+    pub hash: Option<H256>,
+    pub partial_hash: Option<H256>,
+}
```

Comment:
I renamed BlockHeader to BlockHeaderBase because the BorshDeserialize was removed from the BlockHeader. This BlockHeaderBase is inherited by the BlockHeaderLondon and BlockHeaderPreLondon which are not yet used by the client.
When the init happens it fails to Deserialize the header.

Reply:
If you mean that BSC doesn't support London and you need the regular block, just use `BlockHeaderPreLondon`.

Reply:
I updated the code to use BlockHeaderPreLondon